code/modules/spacepods/spacepod.dm

```diff
@@ -73,6 +73,8 @@
 	pr_int_temp_processor = new /datum/global_iterator/pod_preserve_temp(list(src))
 	pr_give_air = new /datum/global_iterator/pod_tank_give_air(list(src))
 	equipment_system = new(src)
+	equipment_system.installed_modules = list()
```

Comment:
`var/list/obj/item/device/spacepod_equipment/installed_modules = list()` should do the exact same thing you've done here, but you don't have to rely on the spacepod New() to do it. You could also just do it in `/datum/spacepod/equipment/New()`.